.github/workflows/ubuntu-focal.yml

```diff
@@ -0,0 +1,42 @@
+name: Ubuntu Focal 20.04 with Ruby 2.x
```

Comment:
The value of the "name" element is also used as the badge image's text. Maybe shorter text is better. See <https://github.com/brianmario/mysql2/actions/workflows/ubuntu-focal.yml/badge.svg>.